src/audio/google/google_ctc_audio_processing_ipc3.c

```diff
@@ -52,9 +52,20 @@ int ctc_set_config(struct processing_module *mod, uint32_t param_id,
 			cd->reconfigure = true;
 		}
 		return 0;
+	case SOF_CTRL_CMD_SWITCH:
```

Comment:
how about getter for switch control?

Reply:
Not really sure what getter do you mean.
Switch control is not going through the config blob.

Reply:
I don't think we need one, its not data that needs to be read out, if there is a case in the future then sure.

Reply:
Thanks! added the getter now.